### ISSUE TITLE
Recommendation: support monitoring software installation process

### ISSUE THREAD

**Issue by PotoTan** [Klocman/Bulk-Crap-Uninstaller#260]
Recommendation: support monitoring software installation process

I am not very familiar with the procedure. 
I saw some uninstall tools claiming that they can monitor the software installation process, so they can use this information to completely uninstall the software when uninstalling

I am not sure whether the function of this monitoring software installation process is better?

**Comment by Klocman** [comment]
Duplicate of #258
I'm currently not planning to implement this feature because of limited time. Even when implemented, it would not have that much of an effect in many cases so it's fairly low return for the time needed to implement.

**Comment by garry-ut99** [comment]
> Klocman: Even when implemented, it would not have that much of an effect in many cases so it's fairly low return for the time needed to implement.

I don't think so, better check your AppData folder (especially its subfolders like Local and Roaming) if you didn't format your os partition for long. I recently discovered that it was containing several tens of thousands of junk files and thousands of folders and subfolders, with a total size of several GB, they were leftovers after previously uninstalled programs through months and years, I purged out all of them (except those who were required by system and currently installed apps), freeing up a lot of space on system partition and giving the system better performance. Unfortunately Bulk-Crap-Uninstaller was unable to find or clean anything in AppData directory and I had to discover these junk files and do the whole cleaning manually myself.

https://www.google.com/search?q=AppData+folder+big

By they way: Revo Uninstaller already has such feature implemented (in the Pro payed version only):
"Real-Time monitoring of system changes & Logs Database"
https://www.revouninstaller.com/products/revo-uninstaller-pro/
